templates/web/prod/WorkZone/Macros.html.twig

```diff
@@ -336,7 +356,11 @@
                              class="ui-accordion-header ui-state-default
                              ui-corner-all header SSTT {% if not basket.getId() %}unread{% endif %}
                              {% if basket.getId() == selected_id and selected_type == 'basket' %}active{% endif %}
-                             {{ classes }}">
+                             {{ classes }}
+                             {# block defined in lib/Alchemy/Phrasea/Helper/WorkZone.php #}
```

Comment:
do not duplicate responsibility. 
- twig restore ux and initiate checkboxes as chkd / unchkd depending on saved preferences. twig doesn't know the effect of ckboxes.
- js display or hide elements.